cognite/client/data_classes/data_modeling/__init__.py

```diff
@@ -131,8 +131,6 @@
     "ContainerProperty",
     "Primitive",
     "CDFExternalIdReference",
-    "RequiresConstraintDefinition",
```

Comment:
We still want to include them, they've just been renamed. I think they're just called RequiresConstraint and UniquenessConstraint.

Reply:
Makes sense - they were missing in the import statement as well 👍